getDataTableReader.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Get Data (TableDataReader)</title>
+
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+
+    <script type="module">
+      // TableauEventType represents the type of Tableau embedding event that can be listened for.
+    //  import { TableauEventType } from "https://public.tableau.com/javascripts/api/tableau.embedding.3.latest.js";
+    // TODO: change import with 22.4 release on Public
+      import { TableauEventType } from 'https://devplat.tableautest.com/javascripts/api/tableau.embedding.3.latest.js'
+
+      async function handleFirstInteractive (e) {
+        const getDataButton = document.getElementById('get-data');
+
+        let pageSelector = $('#pageSelect');
+        pageSelector.prop('disabled', false);
+
+        let pageNumber = '${pageSelector.val()}';
```

Comment:
To use a template literate (`${pageSelector.val()}`), the string needs to be wrapped in backticks, not single quotes:

```
let pageNumber = `${pageSelector.val()}`;
```

However, that will give you the number as a *string* and I think you want it as a number. You should use:

```
const pageNumber = pageSelector.val() || 0;
```

This will default to 0 if `Select page` is selected in the dropdown (since the value is `null` when it's selected).